src/hash.cr

```diff
@@ -833,6 +833,7 @@ class Hash(K, V)
   end
 
   protected def find_entry(key)
+    return nil if empty?
```

Comment:
Please can we have a blank line seperating this check and the definitions below.

Reply:
Done.